src/components/incidents/IncidentFilter.js

```diff
@@ -0,0 +1,40 @@
+import { DateTime } from 'luxon';
+import Interval from 'luxon/src/interval.js';
+
+const newData = arr => {
+  const newCurrentData = arr.map(obj =>
+    Object.keys(obj)
+      .filter(x => obj[x] !== null)
```

Comment:
Same comment as above. I know to you it was just a quick thing to use to get it functional, but that might not be clear to anyone coming behind you to maintain this code.